sopel/modules/wikipedia.py

```diff
@@ -193,7 +200,7 @@ def say_section(bot, trigger, server, query, section):
         return
 
     msg = '{} - {} | "{}"'.format(page_name, section.replace('_', ' '), snippet)
-    bot.say(msg)
+    bot.say(msg, trailing=' [...]"')
```

Comment:
Why not this bad boy here too?

```suggestion
    bot.say(msg, trailing=' […]"')
```

Reply:
Oh, I was matching what it appended before ([`wikipedia.py` L236](https://github.com/sopel-irc/sopel/pull/1995/files#diff-c5ee1fef33221c1015f3a3313648113e1be501376eeea2354c013db3926958c2L236)), but there's no real reason to do so and I already changed the ellipsis style in this PR's `wiktionary.py` patch… I'm just silly? 😁

Reply:
That's why I already approved the PR, I knew you would fix it or have a good reason. ❤️